src/traces/box/calc.js

```diff
@@ -286,7 +286,9 @@ module.exports = function calc(gd, trace) {
                 q1: _(gd, 'q1:'),
                 q3: _(gd, 'q3:'),
                 max: _(gd, 'max:'),
-                mean: trace.boxmean === 'sd' ? _(gd, 'mean ± σ:') : _(gd, 'mean:'),
+                mean: (trace.boxmean === 'sd') || (trace.sizemode === 'sd') ?
+                    _(gd, 'mean ± σ:').replace('σ', trace.sdmultiple === 1 ? 'σ' : (trace.sdmultiple + 'σ')) : // displaying mean +- Nσ whilst supporting translations
```

Comment:
This line is getting too long.
How about adding a`getMean(gd, trace)` and move & revise this logic there?

Reply:
Putting the code into a function would make the code less readable - if readability is the concern here.

Reply:
I agree - if it's only going to be used here, extracting to a function just means it takes another jump to read and understand the code. @archmoj if your concern is just line length we can break this onto more lines. Or leave it for now and sometime add `prettier` to our toolchain like we have in various other projects 😉